DSCResources/MSFT_xHostsFile/MSFT_xHostsFile.psm1

```diff
@@ -0,0 +1,267 @@
+$Script:hostsFilePath = "${env:windir}\system32\drivers\etc\hosts"
+
+Import-LocalizedData -BindingVariable LocalizedData -filename MSFT_xHostsFile.psd1 -BaseDirectory $PSScriptRoot -Verbose
+
+function Get-TargetResource
+{
+    [OutputType([Hashtable])]
+    param (
+        [parameter(Mandatory = $true)]
+        [string]
+        $HostName,
+        [parameter(Mandatory = $true)]
+        [string]
+        $IPAddress
+    )
+
+    $configuration = @{
+        HostName = $HostName
+        IPAddress = $IPAddress
+    }
+
+    Write-Verbose $localizedData.checkingHostsFileEntry
+    try {
```

Comment:
Style point in that we use the style of { on it's own line, I think @PlagueHO has some tricks or tool that makes this easier to fix.